manimlib/animation/transform.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, method, *args, **kwargs):
         method is a method of Mobject, *args are arguments for
         that method.  Key word arguments should be passed in
         as the last arg, as a dict, since **kwargs is for
-        configuration of the transform itslef
+        configuration of the transform itself
```

Comment:
(As I stated elsewhere) Should probably be in a separate general cleanup PR (for organization purposes)

Reply:
I'm fine with doing small typo fixes while in other PRs. If this contained dozens of doc/typo fixes then I'd ask for separate PRs.

Reply:
Actually, I do not know why this change is in my PR as it is not my commit. I pulled from master and created my branch for the matrix changes.